content/en/platform/corda/4.5/enterprise/network/network-map.md

```diff
@@ -99,7 +99,7 @@ the network, along with the network parameters file and identity certificates. G
 online at once - an offline node that isn’t being interacted with doesn’t impact the network in any way. So a test
```

Comment:
I saw that the Package namespace link is fine in this one, but you could change the format to match my suggestion from the other files. Though this is not super important obviously.